exts/cesium.omniverse.cpp.tests/cesium/omniverse/cpp/tests/extension.py

```diff
@@ -14,15 +16,27 @@ def on_startup(self):
         global tests_interface
         tests_interface = acquire_cesium_omniverse_tests_interface()
 
-        tests_interface.on_startup("exts/cesium.omniverse")
+        tests_interface.on_startup(os.path.join(os.path.dirname(__file__), "../../../../../cesium.omniverse"))
```

Comment:
I wonder if there's a more standard way to get an extension's location?

Reply:
It doesn't feel ideal, but this is how the main app handles this. It just doesn't look as awkward there since it's not as deeply nested